classes/EventRegistry.php

```diff
@@ -263,4 +265,67 @@ public function pruneTranslatedContentTemplates($templates)
             return !Str::endsWith($template->getBaseFileName(), $extensions);
         });
     }
+
+    /**
+     * Adds language suffixes to mail view files.
+     * @param  \October\Rain\Mail\Mailer $mailer
+     * @param  \Illuminate\Mail\Message $message
+     * @param  string $view
+     * @param  array $data
+     * @param  string $raw
+     * @param  string $plain
+     * @return string|null
+     */
+    public function findLocalizedMailViewContent($mailer, $message, $view, $data, $raw, $plain)
+    {
+        if (isset($raw) || (!isset($view) && !isset($plain))) {
+            return;
+        }
+
+        $locale = strtolower(isset($data['_current_locale']) ? $data['_current_locale'] : App::getLocale());
+
+        $factory = $mailer->getViewFactory();
+
+        if (isset($view)) {
+            $view = $this->getLocalizedView($factory, $view, $locale);
+        }
+
+        if (isset($plain)) {
+            $plain = $this->getLocalizedView($factory, $plain, $locale);
+        }
+
+        if (($code = $view ?: $plain) === null) {
+            return;
+        }
```

Comment:
What case are you trying to handle here?

Reply:
if no localized html view or plain view is found, we return so the regular view is rendered.